eos/db/saveddata/fit.py

```diff
@@ -106,7 +106,7 @@ def __repr__(self):
                collection_class=HandledProjectedModList,
                cascade='all, delete, delete-orphan',
                single_parent=True,
-               primaryjoin=and_(modules_table.c.fitID == fits_table.c.ID, modules_table.c.projected == True)),
+               primaryjoin=and_(modules_table.c.fitID == fits_table.c.ID, modules_table.c.projected is True)),
```

Comment:
@blitzmann good catch. :)

Yes, it was tested.  Basically each time I modified files I tested what that commit was working on.

In this case, it _did_ fail because, yes, SQL Alchemy doesn't support it.  If you look down the commit list you'll notice I reverted it back to == for exactly the reason you mention:
https://github.com/pyfa-org/Pyfa/pull/774/commits/7cf4134a7347e1412687eeaec2a6de97da23cc5f